bindings/ergo-lib-wasm/src/ast.rs

```diff
@@ -71,6 +72,13 @@ impl Constant {
             .map(I64::from)
     }
 
+    /// Create from byte array(BigInt)
+    pub fn from_byte_array_bigint(num: &[u8]) -> Constant {
```

Comment:
Let's say what format is expected here:
```suggestion
    pub fn from_bigint_signed_bytes_be(num: &[u8]) -> Constant {
```